### DISCUSSION TITLE
How to request access to https://github.com/enterprises/uhg/sso

### DISCUSSION THREAD

**Discussion by m0carthy** [community/community#41742] [Enterprise]
### Select Topic Area

Question

### Body

How do I request access to this GitHub resource...??  https://github.com/enterprises/uhg/sso

I get the following message when I try to access UHG/Vanguard resources in GitHub:

Request Id: cf770d90-a003-4bb2-8045-5c2a3cf97101
Correlation Id: f5574718-40c6-4d9a-b3bd-6c8d607dc45b
Timestamp: 2022-12-14T16:12:37Z
Message: AADSTS50105: Your administrator has configured the application PROD-GitHub-Enterprise-Managed-User ('07fbad4e-6030-484b-be89-c0873f04317f') to block users unless they are specifically granted ('assigned') access to the application. The signed in user 'matthew_mccarthy@optum.com' is blocked because they are not a direct member of a group with access, nor had access directly assigned by an administrator. Please contact your administrator to assign access to this application.

**Comment by getlost01** [comment]
To request access to the GitHub resource at the URL you provided, you will need to follow the steps below:

1. First, make sure you have a GitHub account. If you don't have one, you can sign up for a free account at https://github.com/.
2. Next, navigate to the repository page for the resource you want to access. In this case, the URL you provided is https://github.com/enterprises/uhg/sso, so you should navigate to that page.
3. If the repository is private, you will need to request access in order to view or contribute to the repository. To do this, click on the "Request access" button on the repository page.
4. Fill out the form that appears, providing a brief description of your request and any relevant information about your project or organization.
5. Click "Submit request" to send your request to the repository's owner or maintainer.

The repository owner or maintainer will then review your request and decide whether to grant you access to the repository. If your request is approved, you will receive an email notification and will be able to access the repository. If your request is denied, you will also receive an email notification explaining the decision.

It's worth noting that the repository owner or maintainer has the final say on whether to grant access to the repository, and they may choose to deny your request for any reason. If your request is denied, you may want to try reaching out to the repository owner or maintainer directly to discuss your request further.
